files/es/learn/accessibility/mobile/index.html

```diff
@@ -286,36 +249,32 @@ <h4 id="No_deshabilitar_el_zoom">No deshabilitar el zoom</h4>
 
 <p>Usando viewport, es posible deshabilitar el zoom, usando un código como este en su {{htmlelement("head")}}:</p>
 
-<pre><code>&lt;meta name="viewport" content="user-scalable=no"&gt;</code></pre>
+<pre class="brush: html">&lt;meta name="viewport" content="user-scalable=no"&gt;</pre>
 
 <p>Nunca debe hacer esto si es posible: muchas personas confían en el zoom para poder ver el contenido de su sitio web, por lo que eliminar esta funcionalidad es una muy mala idea. Hay ciertas situaciones en las que el zoom podría romper la interfaz de usuario; en tales casos, si cree que necesita deshabilitar el zoom, debe proporcionar algún otro tipo de equivalente, como un control para aumentar el tamaño del texto de una manera que no rompa su interfaz de usuario.</p>
 
 <h4 id="Mantener_los_menús_accesibles.">Mantener los menús accesibles.</h4>
 
-<p> </p>
-
 <p>Debido a que la pantalla es mucho más estrecha en los dispositivos móviles, es muy común utilizar consultas de medios y otras tecnologías para hacer que el menú de navegación se reduzca a un pequeño icono en la parte superior de la pantalla, que se puede presionar para mostrar el menú solo si es necesario - cuando el sitio se ve en el móvil. Esto suele representarse mediante un icono de "tres líneas horizontales" y, por lo tanto, el patrón de diseño se conoce como "menú de hamburguesas".</p>
 
-<p>Al implementar un menú de este tipo, debe asegurarse de que el control para revelarlo sea accesible mediante los mecanismos de control apropiados (normalmente, toque para dispositivos móviles), como se explica en <a href="#Control mechanisms">Control mechanisms</a>, y que el resto La página se aleja o se oculta de alguna manera mientras se accede al menú, para evitar confusiones al navegar.</p>
+<p>Al implementar un menú de este tipo, debe asegurarse de que el control para revelarlo sea accesible mediante los mecanismos de control apropiados (normalmente, toque para dispositivos móviles), como se explica en <a href="#mecanismos_de_control">Control mechanisms</a>, y que el resto La página se aleja o se oculta de alguna manera mientras se accede al menú, para evitar confusiones al navegar.</p>
```

Comment:
So, should this be like this? 
```suggestion
<p>Al implementar un menú de este tipo, debe asegurarse de que el control para revelarlo sea accesible mediante los mecanismos de control apropiados (normalmente, toque para dispositivos móviles), como se explica en <a href="#mecanismos_de_control">mecanismos de control</a>, y que el resto La página se aleja o se oculta de alguna manera mientras se accede al menú, para evitar confusiones al navegar.</p>
```

Reply:
Applied this.